send-transaction-service/src/send_transaction_service.rs

```diff
@@ -309,10 +335,16 @@ impl SendTransactionService {
         send_socket: &UdpSocket,
         tpu_address: &SocketAddr,
         wire_transaction: &[u8],
-    ) {
+        deduper: &Deduper,
+    ) -> u64 {
+        let dedup_fail = deduper.dedup_buffer(wire_transaction);
```

Comment:
Since this is a full wire transaction buffer, it includes signatures and is vulnerable to signature malleability

Reply:
aren't those signatures invalid?  basically sig fails?

Reply:
Yeah, once we get to sigverify, but STS will still retry them since it's upstream and has no idea.